smarts/sstudio/types.py

```diff
@@ -527,17 +526,23 @@ def pick_remaining_shape_after_split(geometry_collection, expected_point, lane):
 
             # We assume that there are only two splited shapes to choose from
             keep_index = 0
+            print(f"After keep_index = 0")
+            print(lane_shape[1])
+            print(f"This method causes the error: {lane_shape[1].minimum_rotated_rectangle.contains(expected_point)}")
             if lane_shape[1].minimum_rotated_rectangle.contains(expected_point):
```

Comment:
When the `xcb` error occurs, it is always caused by this line. This line does not cause the error most of the time